README.md

```diff
@@ -36,7 +38,22 @@ The `GITHUB_TOKEN` secret is [automatically generated by GitHub](https://help.gi
 
 By default, you probably don’t need to configure `image-actions`. However, if you’d like to ignore specific file paths, or change image compression options, read on.
 
-Change the configuration options by adding a `.github/calibre/image-actions.yml` file:
+Change the configuration options by adding arguments to the action:
+
+```yml
+- name: Compress Images
+  uses: calibreapp/image-actions@master
+  with:
+    githubToken: ${{ secrets.GITHUB_TOKEN }}
+    jpegQuality: '80'
+    pngQuality: '80'
+    webpQuality: '80'
+    ignorePaths: "node_modules/**,build"

```

Comment:
```suggestion
    ignorePaths: "node_modules/**"
```

Reply:
I'm doing it this way, to show the format that multipule entries are put.